stdlib/public/Concurrency/Task.swift

```diff
@@ -379,6 +379,97 @@ struct JobFlags {
   }
 }
 
+// ==== Task Creation Flags --------------------------------------------------
+
+/// Flags for schedulable jobs.
+///
+/// This is a port of the C++ FlagSet.
+@available(SwiftStdlib 5.5, *)
+struct TaskCreateFlags {
+  /// The actual bit representation of these flags.
+  var bits: Int = 0
+
+  /// The priority given to the job.
+  var priority: TaskPriority? {
+    get {
+      let value = Int(bits) & 0xFF
+
+      if value == 0 {
+        return nil
+      }
+
+      return TaskPriority(rawValue: UInt8(value))
+    }
+
+    set {
+      bits = (bits & ~0xFF) | Int(newValue?.rawValue ?? 0)
+    }
+  }
+
+  /// Whether this is a child task.
+  var isChildTask: Bool {
+    get {
+      (bits & (1 << 8)) != 0
+    }
+
+    set {
+      if newValue {
+        bits = bits | 1 << 8
+      } else {
+        bits = (bits & ~(1 << 8))
+      }
+    }
+  }
+
+  /// Whether to copy thread locals from the currently-executing task into the
+  /// newly-created task.
+  var copyThreadLocals: Bool {
```

Comment:
Hmm... Is this `copyThreadLocalTaskLocals` rather?

Reply:
D'oh! Yes, task locals, not thread locals.